neofetch

```diff
@@ -93,6 +93,9 @@ get_distro() {
                     "tiny") distro="${distro//version*}" ;;
                 esac
 
+            elif type -p tazpkg >/dev/null; then
+                distro="SliTaz $(< /etc/slitaz-release)"
```

Comment:
Since it has no other release information, I think we can delete those lines since I don't think they are necessary.